File: 6 Sum Square Difference/problem6.js
sumSquareDifferenceUpTo = function(limit) {
  var regular_sum = 0;
  var squared_sum = 0;
  x = 1;
  while (x <= limit) {
    regular_sum = regular_sum + x;
    squared_sum = squared_sum + Math.pow(x,2);
    x ++;
  }
  return Math.pow(regular_sum,2) - squared_sum;
};

console.log(sumSquareDifferenceUpTo(100));

File: 6 Sum Square Difference/problem6.py
import math

def squared_sum_to(number):
  regular_sum = 0
  squared_sum = 0

  for x in range (1, number + 1):
    regular_sum = regular_sum + x
    squared_sum = squared_sum + math.pow(x,2)

  return math.pow(regular_sum,2) - squared_sum
  
print squared_sum_to(100) 